src/components/input/input.yaml

```diff
@@ -162,3 +166,11 @@ examples:
       id: input-with-autocomplete-attribute
       name: postcode
       autocomplete: postal-code
+  - name: with pattern attribute
```

Comment:
Non-blocking for the future, we tend to only add examples to render if they can't be fully tested using the unit tests.

Reply:
Sorry, sort of! I wanted to confirm it popped up the number keyboard on iOS (to ensure the new attributes hadn't overridden the pattern).

I can remove it again? It works fine.

<img width="568" alt="test number keypad" src="https://user-images.githubusercontent.com/415517/53807336-396a3380-3f47-11e9-8716-7a806f598365.png">

Reply:
Yeah I think it's up to you, just letting you know when we tend to add examples. 👍